validator/src/main/resources/expected-data-template/springboot/springbootAppExpectedHTTPTrace.mustache

```diff
@@ -0,0 +1,37 @@
+[{
+  "name": "/outgoing-http-call",
+  "fault": false,
+  "error": false,
+  "http": {
+    "request": {
+      "url": "{{endpoint}}/outgoing-http-call",
+      "method": "GET"
+    },
+    "response": {
+      "status": 200
+    }
+  },
+  "subsegments": [
+    {
+      "fault": false,
+      "error": false,
+      "subsegments": [
+        {
+          "name": "aws.amazon.com",
+          "fault": false,
+          "error": false,
+          "http": {
+            "request": {
+              "url": "https://aws.amazon.com/",
```

Comment:
if we just want to test the outgoing-http-call, maybe just calling the sample app itself will make the test more stable

Reply:
just got 500 in our canary test. here is the json body we got from xray, you can see status is 500 and error is true

                "subsegments": [
                    {
                        "id": "9d629cbd0d36a450",
                        "name": "aws.amazon.com",
                        "start_time": 1606969615.2203882,
                        "end_time": 1606969615.2429497,
                        "fault": false,
                        "error": true,
                        "http": {
                            "request": {
                                "url": "https://aws.amazon.com",
                                "method": "GET"
                            },
                            "response": {
                                "status": 500,
                                "content_length": 0
                            }
                        },
                        "aws": {
                            "xray": {
                                "auto_instrumentation": false,
                                "sdk_version": "1.1.0",
                                "sdk": "X-Ray for Go for Go"
                            }
                        },
                        "metadata": {
                            "default": {
                                "http.response_content_length": ""
                            }
                        },
                        "namespace": "remote",
                        "subsegments": [
                            {
                                "id": "a35b9f0af395066e",
                                "name": "request",
                                "start_time": 1606969615.2204092,
                                "end_time": 1606969615.2205038,
                                "fault": false,
                                "error": false,
                                "aws": {
                                    "xray": {
                                        "auto_instrumentation": false,
                                        "sdk_version": "1.1.0",
                                        "sdk": "X-Ray for Go for Go"
                                    }
                                }
                            },
                            {
                                "id": "4e70f38632f72310",
                                "name": "response",
                                "start_time": 1606969615.2205071,
                                "end_time": 1606969615.2429152,
                                "fault": false,
                                "error": false,
                                "aws": {
                                    "xray": {
                                        "auto_instrumentation": false,
                                        "sdk_version": "1.1.0",
                                        "sdk": "X-Ray for Go for Go"
                                    }
                                }
                            }
                        ]
                    }
                ]
            },
            "Id": "db8287ae0cb92c4d"
        },

Reply:
That's really interesting that it returns 500, I would expect it to never return 500... Is this avoided if we pick a different endpoint?

Reply:
> if we just want to test the outgoing-http-call, maybe just calling the sample app itself will make the test more stable

Do you mean not including the HTTP call in the App route method? I think that's reasonable for a different route, like even the `/` route, but I think the point of the `/outgoing-http-call` is to make a request to an external service so we should probably keep that validation... although we should resolve this 500 response code issue 😕

Reply:
make sense, then maybe don't validate the error, fault and statuscode key?

Reply:
Do you know why it returns a 500 status code? I don't expect endpoints to ever return 500+ status codes, maybe we should pick a different endpoint?

Reply:
I think we should because that is very essential to test. That will tell us about any kind of throttle or fault  span issues. We must keep those in validation.

Reply:
Then we need to figure out a way, we saw 500 from time to time